packages/expo-cli/src/commands/eject.ts

```diff
@@ -61,6 +55,7 @@ export default function (program: Command) {
       'Create Xcode and Android Studio projects for your app. Use this if you need to add custom native functionality.'
     )
     .helpGroup('eject')
+    .option('--force', 'Skip legacy eject warnings.')
```

Comment:
I think this is not the best use of `--force`, a better use that comes to mind is the ability to skip the git warning, the ability to always generate a new set of native folders, or the ability to force the yarn install to clear before running.

Reply:
this already existed before, we're using `options.force` above for the purpose described here. the description of the command was incorrect before: https://github.com/expo/expo-cli/commit/d7ef4593de17826456bf2b63a0b043feaa070ef3#diff-3fd4eed71d65dfed8c0a6b88b86a23c9c05327ae44ab43fef489e50cc3d7c6e9L69-L72 and i removed it a bit too hastily. i think we should repurpose it at some point to do what you're describing - it probably makes sense to drop it when we no longer support sdk 36 (and therefore no longer support "legacy eject").